test/Components/InputModelTests.re

```diff
@@ -265,6 +263,48 @@ describe("handleInputS#handleInput", ({describe, _}) => {
     });
   });
 
+  describe("When <BS> with no selection", ({test, _}) => {
+    let keys = ["<BS>", "<C-h>"];
+
+    keys
+    |> List.iter(key => {
+         test("Removes character on the left of cursor", ({expect, _}) => {
+           let selection = collapsedSelection(4);
+           let expected = collapsedSelection(3);
+
+           let (text, newSelection) = runInputHandler(selection, key);
+
+           expect.equal(expected, newSelection);
+           expect.string(text).toEqual(
+             "Som interesting. Test. String. Isn't it? Maybe",
+           );
+         });
+
+         test(
+           "Doesn't remove character if cursor at the beginnig",
```

Comment:
```suggestion
           "Doesn't remove character if cursor at the beginning",
```

(I can apparently see typos now, just as long as they're not mine)

Reply:
😆 

We need to bring in a spell-checker extension! Would be a good test case for the exthost